### PR TITLE
update data for winter term 2015/2016

### PR DESCRIPTION
**Pull request by j4yk**
also fixed some of the existing entries

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -344,12 +388,23 @@ var data = {
         dozent: ["Prof. Dr. Andreas Polze", "Frank Feinbube", "Uwe Hentschel"],
         nameLV: "Eingebettete Betriebssysteme"
     },
+    eingebetteteBetriebssystemeWithITSE: {
+        kurz: "Embedded<br />Systems (mit ITSE)",
+        kennung: [IST, OSIS, SAMT, ITSE],
```
Comment (knub) [inline js/data.js]: Kein SAMT.

File: js/data.js [hunk 2]
```diff
@@ -344,12 +388,23 @@ var data = {
         dozent: ["Prof. Dr. Andreas Polze", "Frank Feinbube", "Uwe Hentschel"],
         nameLV: "Eingebettete Betriebssysteme"
     },
+    eingebetteteBetriebssystemeWithITSE: {
+        kurz: "Embedded<br />Systems (mit ITSE)",
```
Comment (knub) [inline js/data.js]: Better `(WS1516)` instead of `(mit ITSE)`.

File: js/data.js [hunk 3]
```diff
@@ -993,8 +1059,19 @@ var data = {
         dozent: ["Dr. Rolf Specht"],
         nameLV: "Personal Leadership Skills"
     },
+    pointCloudAnalytics: {
+        kurz: "Point Cloud<br />Analytics",
+        kennung: [ITSE, HCT, IST],
+        lehrform: ["Seminar"],
+        modul: ["Softskills"],
```
Comment (knub) [inline js/data.js]: Wrong `modul`.

File: js/data.js (hunk 1 above)
Comment (j4yk) [inline js/data.js]: Here it says, SAMT is possible: http://hpi.de/studium/lehrveranstaltungen/it-systems-engineering/lehrveranstaltung/course/2015/eingebettete_betriebssysteme-2.html

File: js/data.js [hunk 4]
```diff
@@ -1048,17 +1125,39 @@ var data = {
         dozent: ["Dr. Matthias Uflacker", "Dipl.-Kfm. Jens Krüger", "Thomas Kowark", "Franziska Häger"],
         nameLV: "Global Team-Based Product Innovation & Engineering I - ME 310"
     },
+    productInnovationEngineeringI_HCT: {
+        kurz: "ME310 I (mit HCT)",
+        kennung: [ITSE, BPET, HCT],
```
Comment (knub) [inline js/data.js]: Can't find this with `HCT` in the current version: https://hpi.de/fileadmin/user_upload/hpi/dokumente/studiendokumente/master/stundenplaene/ws_201516/lv_angebot_master_ws201516.pdf.

File: js/data.js [hunk 5]
```diff
@@ -1421,6 +1542,28 @@ var data = {
...
+    trendsInBioinformatics: {
+        kurz: "Trends in <br />Bioinformatics",
+        kennung: [ITSE, BPET, HCT],
+        lehrform: ["Seminar"],
+        modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
+        semester: [ws15_16],
+        benotet: true,
+        cp: 3,
+        dozent: ["Dr. Matthias Uflacker"],
+        nameLV: "Trends in Betriebssystemen (Forschungsseminar)"
```
Comment (knub) [inline js/data.js]: Wrong name.

File: js/data.js (hunk 1 above)
Comment (knub) [inline js/data.js]: I do not find that information on that site.
Even if, we would need to follow the official document, so for now I think its safer to remove `SAMT`.

Comment (j4yk) [inline js/data.js]: Ok, same thing for ME310 and HCT then. I will fix the mistakes later this evening.

**Comment by knub** [comment]
Alright, thanks.
Website club has some issues with deploying atm, so don't expect it to be available immediately.

**Pull request merged**